lua/rustaceanvim/lsp.lua

```diff
@@ -105,6 +105,14 @@ M.start = function(bufnr)
   ---@type RustaceanLspClientConfig
   local lsp_start_opts = vim.tbl_deep_extend('force', {}, client_config)
   local root_dir = get_root_dir(vim.api.nvim_buf_get_name(bufnr))
+   local function normalize_path(path)
```

Comment:
- Let's move this function to the top level (and give it `---@param` / `---@return` type annotations).
- You will need to run stylua (which is why CI is currently failing).

Reply:
kk let me try. My first pull request.